src/components/20-molecules/list/story.js

```diff
@@ -0,0 +1,63 @@
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+
+export default {
+  title: 'Components/List',
+  decorators: [withKnobs],
+};
+
+export const List = () => {
+  const variant = select('variant', {
+    '[default]': '',
```

Comment:
We said we will not use default for such cases. Reason: if a customer will switch from `ordered` to a bullet list, he has to set variant='' ... that could be weird to him. My last PR implemented that feedback from markus